pyvista/plotting/plotting.py

```diff
@@ -1308,7 +1308,21 @@ def scale(self, scale):
 
     @property
     def camera_position(self):
-        """Return camera position of the active render window."""
+        """Return camera position of the active render window.
+
+        Examples
+        --------
+        Return camera's position and then reposition it.
+
+        >>> mesh = examples.download_bunny_coarse()
+
+        >>> pl = pv.Plotter()
+        >>> pl.add_mesh(mesh, show_edges=True, color="white")
+        >>> pl.add_points(mesh.points, color="red", point_size=10)
+        >>> print(pl.camera_position)
+        >>> pl.camera_position = [(0.02, 0.30, 0.73), (0.02, 0.03, -0.022), (-0.03, 0.94, -0.34)]
+        >>> print(pl.camera_position)
```

Comment:
We need to add what is returned by docstring.
```suggestion
        >>> print(pl.camera_position)
        [(0.02, 0.3, 0.73),
         (0.02, 0.03, -0.022),
         (-0.02999850011249063, 0.9399530035247063, -0.33998300127489384)]
```